SConstruct

```diff
@@ -1472,7 +1475,7 @@ def buildImageCommand( source, target, env ) :
 		svgPath = os.path.abspath( svgFilename ),
 		**substitutions
 	)
-	subprocess.check_call( env["INKSCAPE"] + " " + args, shell = True )
+	subprocess.check_call( inkscapeCmd() + " " + args, shell = True )
```

Comment:
I feel like we may be working around a problem of our own making here. If we pass a list of args instead of a formatted string, maybe we can ditch the workaround?

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4524/commits/47c3b132f51605a01c8ffcb022ec473fe37b8cdd